packages/docusaurus-theme-classic/src/theme/CodeBlock/index.tsx

```diff
@@ -244,9 +244,13 @@ export default ({
                   }
 
                   return (
-                    <div key={i} {...lineProps}>
+                    <div key={i} {...lineProps} aria-hidden="true">
```

Comment:
https://twitter.com/sebsilbermann/status/1290253862272741377

aria hidden true does not look like a good practice to me and this is confirmed by someone

Reply:
`aria-hidden`  used to hide non-interactive elements. This is really not the best approach. Moreover, the code should be indexed for easy search.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute

Reply:
Thanks NikitalT, I have actually made the necessary corrections in the latest commit... Kindly check out and I'd be glad to hear your opinion about it... Thank you...